packages/admin/admin-theme/src/paletteOptions.ts

```diff
@@ -1,26 +1,54 @@
 import { PaletteOptions } from "@mui/material/styles";
 
-import { bluePalette, errorPalette, greenPalette, infoPalette, neutrals, successPalette, warningPalette } from "./colors";
+import { errorPalette, greyPalette, infoPalette, primaryPalette, successPalette, warningPalette } from "./colors";
 
 export const paletteOptions: PaletteOptions = {
-    primary: bluePalette,
-    secondary: greenPalette,
```

Comment:
This results in using MUI's default value for the secondary variant. Are we sure that we're not using it anywhere?

Reply:
I'm currently talking to UX about that

Reply:
@johnnyomair @jamesricky I temporarily readded the green secondary color (using the success palette). So we can merge this PR while UX still thinks about this topic.